src/components/capture/Create.tsx

```diff
@@ -182,10 +174,18 @@ function CaptureCreate() {
 
     const discoversSubscription = (discoverDraftId: string) => {
         setDraftId(null);
-        return startSubscription(
-            supabaseClient.from(
-                `${TABLES.DISCOVERS}:draft_id=eq.${discoverDraftId}`
-            ),
+        jobStatusPoller(
+            supabaseClient
+                .from(TABLES.DISCOVERS)
+                .select(
+                    `
+                    draft_id,
+                    job_status
+                `
+                )
+                .match({
+                    draft_id: discoverDraftId,
+                }),
```

Comment:
Call the new poller and pass it the query you want to run and make sure eventually gets a job status of "success"